composer.json

```diff
@@ -21,7 +21,7 @@
         }
     ],
     "require": {
-        "php": ">=7.2",
```

Comment:
We'll need to leave this at 7.2 because [D8 still supports it](https://www.drupal.org/docs/system-requirements/php-requirements#php_required) and will indefinitely, i.e., 7.3 isn't the minimum required version until D9. Thus we'll default to 7.3 in our `.travis.yml` files, but ORCA needs to at least be installable on 7.2 until D8 is no longer supported.

Reply:
Even if Drupal supports an end-of-life PHP version, I'm not sure that Acquia should. We've already removed it from our hosting platform. But it doesn't matter much to me, I'll change it back.